tokio/src/time/mod.rs

```diff
@@ -96,6 +96,7 @@ mod instant;
 pub use self::instant::Instant;
 
 mod interval;
+pub(crate) use instant::InstantInner;
```

Comment:
I don't love this solution but think it might be the least bad? I played with a few different refactors to deal with [the `Inner` of `Clock` in the `cfg_test_util` feature block using `std::time::Instant`](https://github.com/tokio-rs/tokio/blob/338e13b04baa3cf8db3feb1ba2266c0070a9efdf/tokio/src/time/clock.rs#L85-L88). I'm presume is for testing timeouts and such and really didn't want to touch it. I think using the same feature-flagged type alias might be the most maintainable.

Thoughts?